_data/changelogs/component-card.yml

```
@@ -2,6 +2,12 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a margin bug that causes card media to overlap headers on header-first variants.
```

Just a small re-ordering tweak for clarity. 
```suggestion
    summary: Fixed a margin bug on header-first variants that causes card media to overlap headers.
```

---

Good point! Commited and resolved in [fbc1876](https://github.com/uswds/uswds-site/pull/2227/commits/fbc1876acd3731b2ead6c7daa316789f0d62cee6)

---

Good point! Commited and resolved in [fbc1876](https://github.com/uswds/uswds-site/pull/2227/commits/fbc1876acd3731b2ead6c7daa316789f0d62cee6)